Aikido.Zen.Core/Models/Ip/Blocklist.cs

```diff
@@ -155,6 +183,34 @@ public bool IsIPAllowed(string ip, string endpoint)
             }
         }
 
+        public bool IsIPAllowed(string ip)
+        {
+            _lock.EnterReadLock();
+            try
+            {
+                if (!IPHelper.IsValidIp(ip))
+                {
+                    return true; // Invalid IPs are by default allowed
```

Comment:
This could be dangerous, if the reverse proxy fails and starts sending jiberish suddenly everyone can access site? Would actually return false. (but coordinating in slack with other firewalls)